lib/ansible/module_utils/k8s/raw.py

```diff
@@ -51,6 +49,12 @@
 except ImportError:
     HAS_K8S_CONFIG_HASH = False
 
+try:
+    from openshift.dynamic.apply import apply as k8s_apply
```

Comment:
In the actual implementation I think apply will be another method on the client object, just like patch/replace etc (though a lot of the logic will likely stay in this module)

Reply:
No worries, this PR will remain a WIP until the openshift implementation is finalised